packages/gasket-nextjs/package.json

```
@@ -64,15 +64,15 @@
     "jsdom": "^20.0.0",
     "mocha": "^10.0.0",
     "mock-require": "^3.0.3",
-    "next": "^12.0.8",
+    "next": "^13.1.1",
     "nyc": "^15.1.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "setup-env": "^2.0.0",
     "sinon": "^14.0.0"
   },
   "peerDependencies": {
-    "next": ">=10.2.0 < 13"
+    "next": ">=10.2.0 < 14"
```

Actually, peerDeps are going to need to be narrowed down for now as well.

```suggestion
    "next": ">=10.2.0 <= 13.1.1"
```

Check other package ,too.